### DISCUSSION TITLE
Creation of a small opening

### DISCUSSION THREAD

**Discussion by Jostorrr** [firemodels/fds#13166] [Q&A]
Hello,

I thought that this problem would be easy to solve at first but I have a really hard time finding the solution.
My goal is to create the equivalent of a HOLE (to simulate the exchange of temperature, pressure, gas) but smaller than the size of the mesh. For exemple, with a cubic mesh size 10cm, I would like to create an opening at the bottom of a door that is not 10cm high but only 5mm high.

My best try at the moment is with the function HVAC by using a duct and two nodes (one in the room and one out of the room) but it doesn't really work the same as a HOLE.
To verify this, I created a test in which a fire occurs in the room and :
- in the first cas I put two vents on the wall and an HVAC function with a duct ;
- in the second case I create a HOLE with the same size.
The results are quite different, the hole is extracting a lot of heat while a very small portion of heat is released with the HVAC.

Another problem I faced is that by using the feature TMP_EXTERIOR with a RAMP and increasing the temperature outside of a room, and by using the HVAC duct, there is absolutly no increase in temperature inside the room.

I tried my best to be clear here, excuse me if it is confusing.

Thanks for your help.

**Comment by drjfloyd** [comment]
They are not going to be the same, as they are not the same physcially in the model.  

With HOLE, you get the flow as you are able to resolve it on the grid. On a 10 cm grid you cannot resolve the flow over a 5 mm gap. You would need mutiple grid cells across the opening to resolve the pressure drop with the CFD solver. 

The HVAC solver uses an empircal model for the pressure drop. With appropriately defined losses, you will correctly resolve the flow based on the local pressures on either side of the gap. 

You should read the User's Guide sections on leakage.

**Reply by Jostorrr** [reply]
Thank you for your answer.

I thought there might have been a very easy way to simulate this case that I missed but as you said it's not the same in the model. I will try new tests in order to find the best solution.
